core/services/src/main/java/org/exoplatform/services/cms/impl/Utils.java

```diff
@@ -736,7 +736,7 @@ public static String cleanName(String oldName) {
     if (StringUtils.isEmpty(oldName)) {
       return oldName;
     }
-    return replaceSpecialChars(oldName, "&#*?!@.'\"\t\r\n$\\><:;[]/|");
+    return replaceSpecialChars(oldName, "&#*?!@.'\"\t\r\n$\\><:;[]/|+");
```

Comment:
The character '+' is not among illegal JCR chars, so users are allowed to use it in file names.
Could you find another way to fix the issue without replacing '+' characters